staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -1126,6 +1134,12 @@ type MemoryReservation struct {
 	Limits   v1.ResourceList `json:"limits"`
 }
 
+// ShutdownGracePeriodByPodPriority specifies the shutdown grace period for Pods based on their associated priority class value
+type ShutdownGracePeriodByPodPriority struct {
+	Priority                   int32 `json:"priority"`
```

Comment:
Expected godoc on these fields describing what values they accept, what the values mean, and how they are used (even if that info is already in the overall list field, it belongs here too).